### PR TITLE
[WIP] New Op Test framework

### PR DESCRIPTION
**Pull request by tonyyang-svail**
Move OpTest Framework to be executor based.

TODO:
- [x] Support forward for all Ops, will be done on Oct 23rd
- [ ] Support backward for all Ops, will be done on Oct 25th

### CODE REVIEW

File: paddle/operators/gru_unit_op.cc [hunk 1]
```diff
@@ -54,8 +54,9 @@ class GRUUnitOp : public framework::OperatorWithKernel {
     PADDLE_ENFORCE_EQ(
         weight_width, frame_size * 3,
         "The shape of Weight matrix must be [frame_size, frame_size * 3].");
-    auto bias = Input("Bias");
-    if (bias != framework::kEmptyVarName) {
+    // auto bias = Input("Bias");
```
Comment (reyoung) [inline paddle/operators/gru_unit_op.cc]: Remove them.

File: python/paddle/v2/framework/tests/op_test.py [hunk 2]
```diff
@@ -1,9 +1,13 @@
+import pdb
```
Comment (reyoung) [inline python/paddle/v2/framework/tests/op_test.py]: remove it

File: paddle/operators/gru_unit_op.cc (hunk 1 above)
Comment (tonyyang-svail) [reply paddle/operators/gru_unit_op.cc]: done

File: python/paddle/v2/framework/tests/op_test.py (hunk 2 above)
Comment (tonyyang-svail) [reply python/paddle/v2/framework/tests/op_test.py]: done

**Pull request merged**